examples/with-apollo/lib/with-apollo-client.js

```diff
@@ -0,0 +1,66 @@
+import initApollo from './init-apollo'
+import Head from 'next/head'
+import { getDataFromTree } from 'react-apollo'
+import propTypes from 'prop-types'
+
+export default (App) => {
+  return class Apollo extends React.Component {
```

Comment:
Missing `import React from  'react'`

Reply:
Feel free to send a pull request.

Reply:
My bad, not need to `import React from 'react'` any more. 👍  

https://github.com/zeit/next.js/pull/295/files

Reply:
In this case it should be imported though

Reply:
It's still working without any import. :D (removed all `import React...`) :)))